FutureNorthants/northants-website
# Remove phasebanner, fix double pagemains

### Setup

Requires https://github.com/FutureNorthants/northants-design-system/pull/109
Pull that in via yalc

### Testing
1. View home page and expect no difference
2. View a service landing page and expect no difference other than not having two <main> elements. Try each variation - with hero image, without, with top services, without, with hero image and top services, with alert banner, etc.
3. View other page types - news article, service page and expect no difference
